helm/pms-partner/.gitignore

```diff
@@ -1,2 +1,2 @@
 charts/
-Charts.lock
+Chart.lock
```

Comment:
partner-management-services/deploy/pms
/install.sh
CHART_VERSION=1.3.0-beta.3-develop 
change to 
CHART_VERSION=1.3.x